pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -242,13 +242,20 @@ stdenv.mkDerivation rec {
   preBuild = "cd build";
 
   # Add a ‘qemu-kvm’ wrapper for compatibility/convenience.
-  postInstall = ''
-    install -m755 -D $emitKvmWarningsPath $out/libexec/emit-kvm-warnings
+  postInstall =
+  let
+    copyKvmWarningsPathToOutput = ''
+      install -m755 -D $emitKvmWarningsPath $out/libexec/emit-kvm-warnings
+    '';
```

Comment:
I think it would be clearer (and more idiomatic) if `copyKvmWarningsPathToOutput` was just inlined with `lib.optionalString`.